### ISSUE TITLE
item 1

### ISSUE THREAD

**Issue by sakebaabedin** [sakebaabedin/copilot-test#4]
why ai
https://codewave.com/insights/ai-trending-popular-reasons/
how brain works to create breakthrough applications, to use clear logical reasoning to solve complex problems
computing capacity in parallel far improved than conventional approach
enough evidence to support the fact that problems are solved faster than human brain via artificial nn
brief history of Neural Networks
Machine learning algorithms improve from experience, boosting accuracy
handles complex tasks with impressive precision
save time, cut costs, and innovate. 

data-backed decisions
analyzing customer habits
personalize interactions at scale
automate, speed up work, reduce errors
lowers labor costs by handling repetitive tasks
see trends before they happen
understand what customers think via NLP
flexibility /scalability

ai roles
AI Engineers build and fine-tune AI models, making systems smarter and more efficient in everything from healthcare to finance.
Data Scientists work with huge datasets, spotting trends and helping businesses make better decisions.
AI Ethicists ensure AI is fair and responsible, tackling issues like bias and privacy concerns.
Machine Learning Engineers create algorithms that allow AI to learn and improve on its own.
AI Product Managers connect AI development with business goals, making sure AI solutions solve real problems.
Robotics Engineers design AI-powered robots used in factories, hospitals, and even homes.
Cybersecurity Analysts use AI to detect cyber threats and protect sensitive data from hackers.

ai in VA reVAmp: transform & transcend
what human does, AI does better, smarter, faster
Suppose, we have information on award, address, benefit claim, disability, education
that's a lot of info to dive into
how do we identify unique number that indicates a veteran precisely
can ai detect a pattern that is always integer and that is omnipresent in multiple data sources. that depends on how a human brain perceive information, it's a, extremely strenuous job for a pair of eyes to scan through the data and detect a specific pattern
while a simple regex with ai methodology embedded in it can do it in matter of seconds
EVSS knowledgebase
  glean ->  store  -> apply AI  -> present
  structure the unstructured
  Cluster the similar
  Extract to make decision
build a model
decide on architecture
 patient record and appointment
 filing benefit claim
 locator/mailing address

 https://stackoverflow.com/questions/72255800/how-to-fetch-data-from-an-api-and-save-to-mongodb-using-mongoose
 https://www.mongodb.com/resources/solutions/use-cases/training-machine-learning-models-with-mongodb
 https://medium.com/@1segaladi/nosql-the-intentional-choice-for-generative-ai-7ded8d4031b8
 https://scikit-learn.org/stable/modules/linear_model.html

 
 
case studies/cost analysis
 tools available
	https://riskonnect.com/claims-administration/claims-automation-ai-the-future-of-claims-is-here/
	

 techniques
The most popular AI data mining techniques are Classification, Clustering, Regression analysis, Association rule mining, and Anomaly detection. 

Association rule mining

Association Rule Mining is an important component of data mining. Association Rules Mining is used for finding frequent item set, associations, correlations, or causal structures among sets of items or objects.
Association rule mining is a popular technique for discovering patterns and relationships in large datasets. It can help you find hidden insights, such as customer preferences, product recommendations, or market trends.

Tools for ARM: MLxtend, Orange, or PyFIM in Python, or arules, arulesViz, or rCBA in R. If you prefer a graphical user interface, you can use software applications like Weka, RapidMiner, or KNIME. For cloud-based and web-based solutions, online platforms and services such as BigML, Dataiku, or Google Cloud Platform offer association rule mining.

Medical big data faces many challenges, as follows [11]:
Collecting patient data continuously from different sources, thus leading to the high volume of data.
Medical data are almost unstructured or semi-structured.
Medical data are not clear for everyone.
Handling a huge size of medical data.
Extracting useful information from medical big data.

Clustering

Clustering is an unsupervised machine learning technique designed to group unlabeled examples based on their similarity to each other. (If the examples are labeled, this kind of grouping is called classification.) 

Clustering use cases
Clustering is useful in a variety of industries. Some common applications for clustering:

Market segmentation
Social network analysis
Search result grouping
Medical imaging
Image segmentation
Anomaly detection

Top tools for AI keyword clustering:

SE Ranking
Keyword Insights
Keyword Cupid
SEO Scout
Surfer

Regression analysis

AI regression analysis uses supervised machine learning to predict a continuous output value (like price or temperature) based on input variables by finding complex relationships between them. 

https://pythongeeks.org/regression-techniques-in-artificial-intelligence/

 viability

schema design
 data federation
  what & how
 training set
 commonly used approach

choice of algorithm
api available
speed, cost

usability

use cases
UI
what will be the most useful scenario
where is the biggest gap
mock screen
 cluster identification, health, housing, education, support for beneficiary 
 search through text mining
 generate concept map pertaining to a specific user
 what info is missing to correctly identify a Veteran
 consolidated relevant info
 new way of dealing with large data set

languages, java, python

ease of development
automation
bypass identity with a quick search
search engine

document management
data mining
modernization

recommendation engine
medical history analysis

customer service
QA repository

clustering/categorizing
proof of concept
